cms/djangoapps/contentstore/views/course.py

```diff
@@ -1460,32 +1460,108 @@ def _get_usage_info(store, course, split_tests):
 
             validation_summary = split_test.general_validation_message()
             usage_info[split_test.user_partition_id].append({
-                'label': '{} / {}'.format(unit.display_name, split_test.display_name),
+                'label': u"{} / {}".format(unit.display_name, split_test.display_name),
                 'url': unit_url,
                 'validation': validation_summary.to_json() if validation_summary else None,
             })
         return usage_info
 
     @staticmethod
+    def get_content_groups_usage_info(store, course):
+        """
+        Get usage information for content Group Configurations.
+        """
+        modules = store.get_items(course.id, settings={'group_access': {'$exists': True}})
+
+        return GroupConfiguration._get_content_groups_usage_info(course, modules)
+
+    @staticmethod
+    def _get_content_groups_usage_info(course, modules):
```

Comment:
I'd probably rename the original ``get_usage_info`` to be more specific, as now it sounds like it would return all usage info. Maybe ``get_content_experiment_usage_info``, although the code is inconsistent about naming now.